src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -1,12 +1,18 @@
 package mate.academy.hibernate.relations;
 
 import java.util.List;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
```

Comment:
Ensure that the `hibernate.cfg.xml` file is present in your project. This file is crucial for Hibernate configuration and should be located in the resources directory. Without it, the `SessionFactory` initialization in `HibernateUtil` will fail.